update-imported-docs/update-imported-docs.py

```diff
@@ -195,7 +197,12 @@ def main():
 
     curr_dir = os.path.dirname(os.path.abspath(__file__))
     print("curr_dir {}".format(curr_dir))
-    root_dir = os.path.realpath(os.path.join(curr_dir, '..'))
+
+    if is_running_inside_container():
+        root_dir = os.path.realpath(os.path.join(curr_dir, '..' , 'website'))
```

Comment:
How do we know the string `website` is correct?

Reply:
'website' is being mounted in this part of the run command:
```"${PWD}/../:/website"```

(The aim was to set the website root here)

Reply:
(This has been corrected to setting the path manually)